### PR TITLE
Validate choices that are passed into the cli

### PR DESCRIPTION
**Pull request by bmac**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,11 @@ const { GitUtil, SOURCE_ERROR_STATUS, TARGET_ERROR_STATUS } = require('./utils/g
 const { FileExporter } = require('./utils/fileExporter.js');
 const { FileOutcomeManager } = require('./utils/fileOutcomeManger.js');
 const { createPrManager } = require('./utils/pullRequestManager.js')
+const fs = require('fs')
+
+const choices = fs.readdirSync('./config/implementation/')
+      .map(file => file.replace(/\.json$/, ''))
+      .filter(file => !file.endsWith('-debug'));
```
Comment (nomadtechie) [inline index.js]: Can it be simplified with `file.split('/)[0]` ?

File: index.js [hunk 2]
```diff
@@ -19,6 +24,7 @@ const argv = yargs
     demandOption: true,
     describe: 'Specify implementor engine...options are jsc',
```
Comment (nomadtechie) [inline index.js]: do we need to update the describe block? 
options are `${...choices}`

Comment (bmac) [reply index.js]: Good catch.

**Comment by bmac** [comment]
Updated

**Pull request merged**